modules/gathering-data-specific-features.adoc

```diff
@@ -14,6 +20,68 @@ You can gather debugging information about specific features by using the
 The `must-gather` tool supports multiple images, so you can gather data about
 more than one feature by running a single command.
 
+ifdef::from-main-support-section[]
+
+ifndef::openshift-origin[]
+
+.Supported must-gather images
+[cols="2,2",options="header"]
+|===
+|Image |Purpose
+
+|`registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8`
+|Data collection for {VirtProductName}.
+
+|`registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8`
+|Data collection for OpenShift Serverless.
+
+|`registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7`
+|Data collection for Red Hat OpenShift Service Mesh.
+
+|`registry.redhat.io/rhcam-1-2/openshift-migration-must-gather-rhel8`
+|Data collection for migration-related information.
+
+|`registry.redhat.io/ocs4/ocs-must-gather-rhel8`
+|Data collection for Red Hat OpenShift Container Storage.
+
+|===
+
+endif::openshift-origin[]
+
+ifdef::openshift-origin[]
+
+.Available must-gather images
+[cols="2,2",options="header"]
+|===
+|Image |Purpose
+
+|`quay.io/kubevirt/must-gather`
```

Comment:
@warrenvw  Can you please confirm that these images/descriptions listed are correct for serverless/kubevirt? Thanks!

Previews:
* (OCP) https://bz-1818830--ocpdocs.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data.html#gathering-data-specific-features_gathering-cluster-data
* (OKD) https://bz-1818830--ocpdocs.netlify.app/openshift-origin/latest/support/gathering-cluster-data.html#gathering-data-specific-features_gathering-cluster-data